app/controllers/trn_requests_controller.rb

```diff
@@ -7,26 +7,19 @@ def show
     session[:form_complete] = true
   end
 
-  def update # rubocop:disable Metrics/AbcSize
+  def update
     redirect_to root_url unless trn_request
     session[:form_complete] = false
 
     update_trn_request
-    unless FeatureFlag.active?(:use_dqt_api)
-      ZendeskService.create_ticket!(trn_request)
-      TeacherMailer.information_received(trn_request).deliver_now
-      redirect_to helpdesk_request_submitted_url
-      return
-    end
 
     begin
-      response = DqtApi.find_trn!(trn_request)
-      trn_request.update(trn: response['trn'])
-      TeacherMailer.found_trn(trn_request).deliver_now
+      find_trn_using_api
+
       redirect_to trn_found_path
     rescue DqtApi::ApiError, Faraday::ConnectionFailed, Faraday::TimeoutError, DqtApi::TooManyResults
-      ZendeskService.create_ticket!(trn_request)
-      TeacherMailer.information_received(trn_request).deliver_now
+      create_zendesk_ticket
+
       redirect_to helpdesk_request_submitted_url
```

Comment:
I think I previously was against writing something like this in a previous PR, but I think actually the thing that was irking me was bundling up the `redirect_to` inside the helper method as well. I think the control flow methods that are part of the controller should stay at the top level and be visible when just viewing the methods themselves, without drilling into helpers. Either way, curious if this makes sense.

Reply:
Makes sense to me.

Reply:
I think you're right about keeping the redirects in the action rather than the helper methods.